test/e2e/README.md

```diff
@@ -0,0 +1,34 @@
+# GetEnvoy End-to-end (e2e) tests
```

Comment:
@mathetake I restored the old project structure, but intentionally am still not sharing files. Here's why. Hope it makes sense

Reply:
make sense